.flintci.yml

```diff
@@ -1,4 +0,0 @@
-services:
```

Comment:
Do we need to remove flintci?

Reply:
As quick thinking, I prefer https://github.com/symfony/maker-bundle/blob/master/composer.json#L16-L21 than space version "pushed by" flintCi
And https://github.com/symfony/maker-bundle/blob/master/src/Resources/config/services.xml#L3-L5 than inline version "pushed by" flintCi

I don't have strong opinion about flintCi right now (and maybe flintCi has custom option)